scripts/ci/check_compliance.py

```diff
@@ -582,9 +582,11 @@ def check_no_undef_outside_kconfig(self, kconf):
         # Warning: Needs to work with both --perl-regexp and the 're' module
         regex = r"\bCONFIG_[A-Z0-9_]+\b(?!\s*##|[$@{*])"
 
-        # Skip doc/releases, which often references removed symbols
+        # Skip doc/releases and doc/security/vulnerabilities.rst, which often
+        # reference removed symbols
         grep_stdout = git("grep", "--line-number", "-I", "--null",
                           "--perl-regexp", regex, "--", ":!/doc/releases",
+                          ":!doc/security/vulnerabilities.rst",
```

Comment:
just looking at the other path, shouldn't this be: 
```suggestion
                          ":!/doc/security/vulnerabilities.rst",
```

Reply:
Changed, re-verified in https://github.com/zephyrproject-rtos/zephyr/pull/69460.